<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>-1</y>
      <width>721</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideRight</enum>
    </property>
    <property name="usesScrollButtons">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>主界面</string>
     </attribute>
     <widget class="QLabel" name="lb_main_bk">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>711</width>
        <height>381</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_Title">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>30</y>
        <width>171</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Unicode MS</family>
        <pointsize>38</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_user">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>125</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lb_main_user">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>130</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>用户名：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_pwd">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>170</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="lb_main_pwd">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>175</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>密码：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_login">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>210</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>登录</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_main_slogon">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>120</y>
        <width>171</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Unicode MS</family>
        <pointsize>38</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_register">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>210</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>注册</string>
      </property>
     </widget>
     <zorder>lb_main_bk</zorder>
     <zorder>label_Title</zorder>
     <zorder>lb_main_user</zorder>
     <zorder>lb_main_pwd</zorder>
     <zorder>pb_login</zorder>
     <zorder>label_main_slogon</zorder>
     <zorder>le_pwd</zorder>
     <zorder>le_user</zorder>
     <zorder>pb_register</zorder>
     <zorder>pb_login</zorder>
     <zorder>pb_register</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>图书查询</string>
     </attribute>
     <widget class="QLabel" name="lb_search_title">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>10</y>
        <width>171</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Unicode MS</family>
        <pointsize>38</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="text">
       <string>图书查询</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>查询方式：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_search_option">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>90</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_search_keyword">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>90</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tw_search_result">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>140</y>
        <width>721</width>
        <height>241</height>
       </rect>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>8</number>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
     <widget class="QLabel" name="lb_search_bk">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-15</y>
        <width>721</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_search">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>90</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
     <zorder>lb_search_bk</zorder>
     <zorder>lb_search_title</zorder>
     <zorder>label_3</zorder>
     <zorder>cb_search_option</zorder>
     <zorder>le_search_keyword</zorder>
     <zorder>tw_search_result</zorder>
     <zorder>pb_search</zorder>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>图书借还</string>
     </attribute>
     <widget class="QLabel" name="lb_return_title">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>0</y>
        <width>171</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Unicode MS</family>
        <pointsize>38</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="text">
       <string>图书借还</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>80</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>输入书号：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_return_bookid">
      <property name="geometry">
       <rect>
        <x>292</x>
        <y>80</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_return_search">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>80</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_return_b">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>320</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>借书</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_return_r">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>320</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>还书</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_book_return_bk">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-15</y>
        <width>721</width>
        <height>401</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>150</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>书名：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_bookname">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>150</y>
        <width>351</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_author">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>210</y>
        <width>301</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>210</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>作者：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>270</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>状态：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_status">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>270</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <zorder>lb_book_return_bk</zorder>
     <zorder>lb_return_title</zorder>
     <zorder>label_4</zorder>
     <zorder>le_return_bookid</zorder>
     <zorder>pb_return_search</zorder>
     <zorder>pb_return_b</zorder>
     <zorder>pb_return_r</zorder>
     <zorder>label</zorder>
     <zorder>label_bookname</zorder>
     <zorder>label_author</zorder>
     <zorder>label_2</zorder>
     <zorder>label_5</zorder>
     <zorder>label_status</zorder>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>用户管理</string>
     </attribute>
     <widget class="QLabel" name="lb_return_title_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>10</y>
        <width>171</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Unicode MS</family>
        <pointsize>38</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="text">
       <string>账户管理</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_change_info">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>100</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cochin</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>修改密码</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_history">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>150</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cochin</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>借阅记录</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_logout">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>200</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cochin</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>退出登录</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_user_bk">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-15</y>
        <width>721</width>
        <height>401</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>lb_user_bk</zorder>
     <zorder>lb_return_title_2</zorder>
     <zorder>pb_change_info</zorder>
     <zorder>pb_history</zorder>
     <zorder>pb_logout</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
